core/tools/messagehandler/messagemodel.h

```diff
@@ -57,7 +57,7 @@ class MessageModel : public QAbstractTableModel
     virtual QVariant data(const QModelIndex &index, int role = Qt::DisplayRole) const;
     virtual QVariant headerData(int section, Qt::Orientation orientation,
                                 int role = Qt::DisplayRole) const;
-
+    Backtrace getBacktrace(int idx);
```

Comment:
please remove the "get" prefix, just name it "backtrace" following the Qt API naming style.